reframework/autorun/ModOptionsMenu/ModMenuApi.lua

```diff
@@ -127,6 +154,8 @@ local function GetOptionData(mod, optType, label, toolTip, defaultValue, immedia
 	if not data then
 	
 		if not defaultValue then defaultValue = 0; end
+
+		toolTip = WrapText(toolTip);
```

Comment:
You shouldnt do this here since right below the "data.message" should be the original unmodified tooltip that can be checked for changes later. just assign the WrapText directly to the data.displayMessage and it should be good.
Everything else looks good so fix that and I'll merge it in.

Reply:
Oh yeah, true. I did that for options.enumMessages but forgot about this one.

Reply:
Hold on, I think this may cause a bug if the tooltip is updated:
![image](https://user-images.githubusercontent.com/44688128/183312022-daf89fc3-5b42-4d4b-9927-cc0ddd02c8f4.png)
It's updating message, but not displayMessage here.

Reply:
I think it would be best if I made a new property called originalTooltip to store the comparison value.

Reply:
Did a test here: I'd change the tooltip passed to a toggle every time it changed.
The way CheckLabel is right now, it wouldn't update at all since it's not changing opt.displayMessage.
By changing it to this, it started changing correctly:
![image](https://user-images.githubusercontent.com/44688128/183312297-cc778b8f-12e5-4dce-87e8-01285d6bd6c6.png)

Reply:
ah yeah, you should be able to just update the displayMessage in the checklabel thing, forgot about that.

Reply:
Done!

Reply:
Oh, wait one second, I need to change something

Reply:
There we go. I had placed a similar bug to CheckLabel's in the Options enum verification.

Reply:
Yeah good catch, I kinda don't like having to detect and update all these things individually. I almost wonder if it'd be better to just regen everything every time something changes to unify these systems. It'd be less efficient but meh. Already have to regenerate them when you change the layout anyway, and it isn't often you would just change the tooltip or options individually.